### ISSUE TITLE
Upgrading to WKWebView

### ISSUE THREAD

**Issue by opeeters** [irceline/aq-mobile-be#144]
After upgrading to iOS `12.2`, iPhone users were not able to use the app anymore. XCode versions prior to 10.2 did not signal that `UIWebView` would become unusable. Error message since XCode 10.2:

```
2019-03-28 01:27:31.622944+0100 BelAir[12936:1026917] Apache Cordova native platform version 5.0.0 is starting.
2019-03-28 01:27:31.623533+0100 BelAir[12936:1026917] Multi-tasking -> Device: YES, App: YES
2019-03-28 01:27:31.664550+0100 BelAir[12936:1026917] Using UIWebView
2019-03-28 01:27:31.668253+0100 BelAir[12936:1026917] [CDVTimer][console] 0.108004ms
2019-03-28 01:27:31.668464+0100 BelAir[12936:1026917] [CDVTimer][handleopenurl] 0.082016ms
2019-03-28 01:27:31.669990+0100 BelAir[12936:1026917] [CDVTimer][intentandnavigationfilter] 1.438975ms
2019-03-28 01:27:31.670180+0100 BelAir[12936:1026917] [CDVTimer][gesturehandler] 0.061989ms
2019-03-28 01:27:31.670271+0100 BelAir[12936:1026917] [CDVTimer][TotalPluginStartup] 2.204061ms
2019-03-28 01:27:33.050944+0100 BelAir[12936:1026917] Resetting plugins due to page load.
2019-03-28 01:27:33.083677+0100 BelAir[12936:1026917] Failed to load webpage with error: The operation couldn’t be completed. (NSURLErrorDomain error -999.)
2019-03-28 01:27:33.085148+0100 BelAir[12936:1026917] Resetting plugins due to page load.
2019-03-28 01:27:34.101311+0100 BelAir[12936:1027768] NSURLConnection finished with error - code -1100
2019-03-28 01:27:34.102316+0100 BelAir[12936:1027765] NSURLConnection finished with error - code -1100
2019-03-28 01:27:34.104519+0100 BelAir[12936:1027768] NSURLConnection finished with error - code -1100
2019-03-28 01:27:34.105459+0100 BelAir[12936:1027768] NSURLConnection finished with error - code -1100
2019-03-28 01:27:34.107885+0100 BelAir[12936:1027765] NSURLConnection finished with error - code -1100
2019-03-28 01:27:34.108578+0100 BelAir[12936:1027766] NSURLConnection finished with error - code -1100
2019-03-28 01:27:34.120011+0100 BelAir[12936:1026917] Finished load of: file:///Users/patatiepatata/Library/Developer/CoreSimulator/Devices//data/Containers/Bundle/Application/{uuid}/BelAir.app/www/index.html
```

Upgrading `UIWebView` to `WKWebView`:
```
ionic cordova plugin add cordova-plugin-ionic-webview --save
```

See more info:
* https://ionicframework.com/docs/v3/wkwebview/

There might still be some additional steps to complete the upgrade.

**Comment by janschulte** [comment]
As mentioned in a previous mail, this works from our point. Or do you see here other issues?

**Comment by opeeters** [comment]
Ok, let's close the issue..

fixed with 90702cbb08f9586eebe13f4f58256cf3a523642f and 4f7dab08db422e920b66f3ef8ec288fab0ad5644
